src/TrackGenerator.cpp

```diff
@@ -261,21 +282,34 @@ FP_PRECISION* TrackGenerator::getFSRVolumes() {
   FP_PRECISION* FSR_volumes = new FP_PRECISION[num_FSRs];
   memset(FSR_volumes, 0., num_FSRs*sizeof(FP_PRECISION));
 
-  int azim_index;
-  segment* curr_segment;
-  FP_PRECISION volume;
-
   /* Calculate each FSR's "volume" by accumulating the total length of *
    * all Track segments multipled by the Track "widths" for each FSR.  */
-  for (int i=0; i < _num_azim; i++) {
-    for (int j=0; j < _num_tracks[i]; j++) {
+#pragma omp parallel
+  {
 
-      azim_index = _tracks[i][j].getAzimAngleIndex();
+    int azim_index, fsr_id;
+    segment* curr_segment;
+    FP_PRECISION volume;
 
-      for (int s=0; s < _tracks[i][j].getNumSegments(); s++) {
-        curr_segment = _tracks[i][j].getSegment(s);
-        volume = curr_segment->_length * _azim_weights[azim_index];
-        FSR_volumes[curr_segment->_region_id] += volume;
+    for (int i=0; i < _num_azim; i++) {
+#pragma omp for
```

Comment:
All of the OpenMP pragmas show up without any indentation. Is this intentional? They are indented inline within rest of the codebase. I've seen this style elsewhere and honestly do not strongly prefer one style over another, but would like to keep things consistent.